lib/elixir_search_extractor_web/templates/shared/_app-toast.html.eex

```diff
@@ -0,0 +1,15 @@
+<div class="app-toast">
+	<%= unless is_nil(get_flash(@conn, :info)) do %>
+		<div class="alert alert-info fade show app-toast__alert">
+			<p class="app-toast__message"><%= get_flash(@conn, :info) %></p>
+			<a class="app-toast__close" data-bs-dismiss="alert" aria-label="close">&times;</a>
+		</div>
+	<% end %>
+
+	<%= unless is_nil(get_flash(@conn, :error)) do %>
+	<div class="alert alert-danger fade show app-toast__alert">
+		<p class="app-toast__message"><%= get_flash(@conn, :error) %></p>
+		<a class="app-toast__close" data-bs-dismiss="alert" aria-label="close">&times;</a>
+	</div>
+	<% end %>
```

Comment:
Let's fix the indentation here 😆

Reply:
@rosle  Fixed issues in [337cc03](https://github.com/rafayet-monon/elixir_search_extractor/pull/29/commits/337cc03a2eb69e1a0c06ede52357ba4c3fdff9fa)